app/app/urls.py

```diff
@@ -160,10 +160,10 @@
     # images
     re_path(r'^funding/embed/?', dashboard.embed.embed, name='embed'),
     re_path(r'^funding/avatar/?', avatar.views.handle_avatar, name='avatar'),
-    re_path(r'^static/avatar/(.*)/(.*)?', avatar.views.handle_avatar, name='org_avatar'),
```

Comment:
why were these changed?  

the avatar / profile pic lookups are among the most comment requests we get... moving it out of the `/static/` dir is going to massively increase the number of these requests that hit the main app server

Reply:
@owocki I updated the cloudfront caching entries from `/static` to `/dynamic` since we're no longer serving static from the same cloudfront distribution.  We shouldn't see any change on this endpoints beyond the initial surge to re-cache the assets as necessary.